examples/src/main/java/com/teamdev/jxbrowser/examples/ScreenSharing.java

```diff
@@ -60,29 +67,72 @@ public static void main(String[] args) {
             tell.selectSource(screen, AudioCaptureMode.CAPTURE);
         });
 
-        // Subscribe to the capture session start event.
-        browserStreamer.on(CaptureSessionStarted.class, event ->
+        invokeLater(() -> {
+            JFrame frame = new JFrame("Screen Sharing Example");
+            JButton share = new JButton("Share My Screen...");
 
-            // Navigate to the screen sharing URL in the second browser.
-            browserSpectator.navigation().loadUrl(WEBRTC_SCREEN_SHARING_URL)
-        );
+            share.addActionListener(e -> {
 
-        initBrowserView(browserStreamer);
-        initBrowserView(browserSpectator);
+                // Load the WebRTC Screen Sharing Demo
+                // and wait until the document in the main frame is loaded completely.
+                loadUrlAndWaitForDocument(browser, WEBRTC_SCREEN_SHARING_URL);
 
-        browserStreamer.navigation().loadUrl(WEBRTC_SCREEN_SHARING_URL);
-    }
+                // Click the "Share Your Screen" button to start a capture session.
+                browser.mainFrame().ifPresent(mainFrame ->
+                        mainFrame.executeJavaScript(
+                                "document.getElementById('share-screen').click();"));
 
-    private static void initBrowserView(Browser browser) {
-        invokeLater(() -> {
-            BrowserView view = BrowserView.newInstance(browser);
+                showSuccessScreenSharingDialog(frame, browser.url());
+            });
 
-            JFrame frame = new JFrame("Screen Sharing Example");
             frame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
-            frame.add(view, BorderLayout.CENTER);
-            frame.setSize(1200, 800);
+            frame.setSize(500, 300);
+            frame.setLayout(new GridBagLayout());
+            frame.add(share, new GridBagConstraints());
             frame.setLocationRelativeTo(null);
             frame.setVisible(true);
         });
     }
+
+    private static void loadUrlAndWaitForDocument(Browser browser, String url) {
+        CountDownLatch documentLoaded = new CountDownLatch(1);
+        Subscription subscription = browser.navigation().on(FrameDocumentLoadFinished.class, e -> {
+            String currentUrl = e.frame().browser().url();
+            if (currentUrl.startsWith(url)) {
+                documentLoaded.countDown();
+            }
+        });
+        browser.navigation().loadUrlAndWait(url);
+        try {
+            documentLoaded.await();
+        } catch (InterruptedException e) {
+            throw new RuntimeException(e);
+        }
+        subscription.unsubscribe();
+    }
+
+    private static void showSuccessScreenSharingDialog(JFrame frame, String url) {
+        invokeLater(() -> {
+            String title = "You are sharing the primary screen";
+            String message = String.format(
+                    "Please share and open the following URL in Google Chrome to see your screen remotely:%n%s",
```

Comment:
```suggestion
                    "Please open the following URL in Google Chrome to see your screen remotely:%n%s",
```

Does it also work in Firefox?